tests/views/test_collection_exercise.py

```diff
@@ -91,10 +92,15 @@ def test_upload_collection_instrument(self, mock_request):
             'ciFile': (BytesIO(b'data'), '064_201803_0001.xlsx'),
             'load-ci': '',
         }
+        survey_data = {
+            "id": "af6ddd8f-7bd0-4c51-b879-ff4b367461c5"
+        }
```

Comment:
survey_data could be made a class attribute, it is defined identically in several tests.